lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -143,15 +144,20 @@ Function Choco-Upgrade
         [Parameter(Mandatory=$false, Position=9)]
         [bool]$ignoredependencies,
         [Parameter(Mandatory=$false, Position=10)]
-        [string]$executiontimeout
+        [int]$timeout
     )
 
     if (-not (Choco-IsInstalled $package))
     {
         throw "$package is not installed, you cannot upgrade"
     }
 
-    $cmd = "$executable upgrade -dv -y $package"
+    $cmd = "$executable upgrade -dv -y $package -timeout $timeou -timeout $timeout"
```

Comment:
looks like a copy paste problem here?
Should this be 
$cmd = "$executable upgrade -dv -y $package -timeout $timeout"
instead of
$cmd = "$executable upgrade -dv -y $package -timeout $timeou -timeout $timeout"
?